README.md

```diff
@@ -243,19 +243,51 @@ Available targets:
 ```
 <!-- markdownlint-restore -->
 <!-- markdownlint-disable -->
+## Windows Managed Node groups
```

Comment:
If this is manually added, it will be overwritten. You have to add it to README.yaml

Reply:
oh I see what you did with `windows.md`